config/redfish/config.go

```diff
@@ -0,0 +1,123 @@
+// SPDX-FileCopyrightText: 2025 The Kepler Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package redfish
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"gopkg.in/yaml.v3"
+)
+
+// BMCConfig represents the configuration structure for BMC connections
+type BMCConfig struct {
+	Nodes map[string]string    `yaml:"nodes"` // Node name -> BMC ID mapping
```

Comment:
```suggestion
	Nodes map[string]string    `yaml:"nodes"` // Node ID -> BMC ID mapping
```

Reply:
```
kepler_node_cpu_joules_total{node_name="",path="/sys/class/powercap/intel-rapl:0",zone="package"} 74074.275518
```
node_name is the current usage. I think we should just stick with that .. what say?